docs/manual/scala/guide/services/TestingServices.md

```diff
@@ -60,6 +60,8 @@ The server is by default running with [[pubsub|PubSub]], [[cluster|Cluster]] and
 
 If your service needs [[persistence|PersistentEntity]] you will need to enable it explicitly. This can be done by enabling Cassandra or JDBC, depending on which kind of persistence is used by your service. In any case, Lagom persistence requires clustering, so when when enabling one or another, cluster will also be enabled automatically.
```

Comment:
```suggestion
If your service needs [[persistence|PersistentEntity]] you will need to enable it explicitly. This can be done by enabling Cassandra or JDBC, depending on which kind of persistence is used by your service. In any case, Lagom persistence requires clustering, so when enabling one or another, a cluster will also be enabled automatically.
```